cleanlab/classification.py

```diff
@@ -120,7 +120,10 @@ def score(self, X, y, sample_weight=None):
 import pandas as pd
 import inspect
 import warnings
-from typing import TypeVar, Optional
+from typing import Optional, TYPE_CHECKING
+
+if TYPE_CHECKING:
```

Comment:
refactor(classification): 💡 exclude non-executable type-checking code from test coverage report

Code within if TYPE_CHECKING is not executed during runtime, so it won't be covered by tests.

```suggestion
if TYPE_CHECKING:  # pragma: no cover
```

Reply:
Good call, thanks!